end-to-end-tests/tests/tests.robot

```diff
@@ -23,7 +22,7 @@ User can access the survey with valid email and complete survey
     Answer all questions
     Click answer summary button
     Click go to results
-    [Teardown]    Close Browser
+    [Teardown]    Close Application
 
 # tests that database is reseted between tests
 # database reseting not working yet, uncomment assertion when db reset works
```

Comment:
This comment could be deleted since db reset should work now.

Reply:
i'm aware of this but decided not to over-optimize the code yet to get working on CI.